### PR TITLE
Implement Pet Controller API Endpoints

### PR DESCRIPTION
**Pull request by Chris-Johnston**
Implements the endpoints for the Pet Controller. Supports getting by ID, updating, deleting, inserting.

Add support for LESS in the gulp build pipeline.

Adds a page for testing the Pet api endpoints. This page is somewhat incomplete, but as of right now is not a high priority.

The following are not working, **but will be addressed in future PRs**
- [ ] PetTester insert, update is not working.
- [ ] Checkbox 'is dead' is not working
- [ ] no way to pick dates
- [ ] long page gets covered by the sticky footer, still.

### CODE REVIEW

**Comment by Chris-Johnston** [comment]
Ready for review. This can be tested either using the PetTester page, or using Postman. The pet tester page isn't totally feature complete, but isn't high enough priority that I should keep getting blocked by it.

**Comment by Chris-Johnston** [comment]
Once this PR is merged, then the `ActivityController` work can be started, because ideally the endpoints for Activities should be `/api/Pet/{petid}/activities/{activitytype}` to get all recent activities, optionally of a specific type. In addition the `GET /api/Pet/{petid}/status` endpoint could be determined after this is merged too.

**Review by anishprasad01 — APPROVED** [review]

**Pull request merged**
